tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/SwaggerApiParser.csproj

```diff
@@ -11,6 +10,7 @@
     <VersionPrefix>1.0.0</VersionPrefix>
     <AssemblyName>Azure.Sdk.Tools.SwaggerApiParser</AssemblyName>
     <SignAssembly>True</SignAssembly>
+    <InformationalVersion>$(OfficialBuildId)</InformationalVersion>
```

Comment:
This should be getting pulled in from https://github.com/Azure/azure-sdk-tools/blob/main/eng/Versioning.props.

Reply:
I don't know what you mean by this statement.

From experience with test-proxy, the addition of informational version merely means [you can access this](https://github.com/Azure/azure-sdk-tools/blob/d9c34d988342f7dab916d27fc2ebb62aa67f1a70/tools/test-proxy/Azure.Sdk.Tools.TestProxy/Startup.cs#L127) and have it come through with the full version. If you use the dll version with, you always get `1.0.0` and that's it. I can dig up the old code but it's been a while since I updated this. Will do so once I understand more about your question.

Reply:
I just did a bit of digging to confirm. Could be user error but there's a reason that I did this.

![image](https://user-images.githubusercontent.com/45376673/193368838-995144ef-4740-46bc-b13a-2e7c3f626a70.png)

Reply:
I guess I'm unclear what you are trying to fix here, based on the description I thought you were trying to fix the package version because it cannot publish more than once and as such the versioning.targets, assuming the are being used correctly, should already add the officialbuidId.

The InformationVersion is just a string in the dll which I don't think will help with package publishing.

Reply:
You're right. The actual bugfix is to remove the manual packageVersion from the csproj. That's still part of this PR.

By removing the manual setting, we fall back to the common build props, which is how the package will be auto versioned by the buildnumber format etc.